public/js/app.js

```diff
@@ -0,0 +1,34 @@
+'use sctrict';
+$(document).ready(function(){
+  $('.screenshot-gallery').slick({
+    centerMode: true,
+    centerPadding: '60px',
+    slidesToShow: 3,
+    responsive: [
+      {
+        breakpoint: 768,
+        settings: {
+          arrows: false,
+          centerMode: true,
+          centerPadding: '40px',
+          slidesToShow: 3
+        }
+      },
+      {
+        breakpoint: 480,
+        settings: {
+          arrows: false,
+          centerMode: true,
+          centerPadding: '40px',
+          slidesToShow: 1
+        }
+      }
+    ]
+  });
+  $('.review-gallery').slick({
+    dots: true,
+    arrows: false,
+    autoplay: true,
+    autoplaySpeed: 10000
+  });
+});
```

Comment:
Should always have a line after the final line of code. This should be a preference setting in your editor.